package.json

```diff
@@ -30,17 +30,17 @@
   },
   "devDependencies": {
     "@rollup/plugin-typescript": "^8.5.0",
-    "@types/chai": "^4.2.21",
-    "@types/jest": "^27.0.1",
-    "chai": "^4.3.4",
+    "@types/chai": "^4.3.3",
+    "@types/jest": "^28.1.1",
+    "chai": "^4.3.6",
     "esbuild": "^0.15.7",
-    "jest": "^27.1.0",
+    "jest": "^28.1.3",
     "prettier": "^2.7.1",
     "prettier-plugin-organize-imports": "^3.0.3",
     "rollup": "^2.79.1",
     "rollup-plugin-dts": "^4.2.2",
     "rollup-plugin-esbuild": "^4.10.1",
-    "ts-jest": "^27.0.5",
+    "ts-jest": "^28.0.8",
```

Comment:
These package upgrades are awesome 👍 so awesome I copied them into a separate PR and merged it so #118 can use jest's v28+ fake timer features.

Sorry for the conflict, can you rebase off master?

Reply:
Done, thanks!